setup.cfg

```diff
@@ -31,6 +31,7 @@ universal = 1
 universal = 1
 
 [flake8]
+ignore = D100,D101,D102,D103,D104,D105,D106,D107
```

Comment:
Not a fan of ignoring these. We should either collectively decide to remove flake8-docstrings or actually implement docstrings. You already comment quite extensively, so why not put that in docstrings where it can be inspected rather than in the comments?

Reply:
This is about not requiring a docstring for every single public package, module, class and method. We are writing applications, not libraries, and in some cases, a docstring can simply be unnecessary, redundant and noisy.

I am still in favor of enforcing conventions where docstrings _are_ used, and I think whether a docstring is or isn't necessary is a matter of judgement by developers and during code review.